docs/glossary.md

```diff
@@ -114,7 +114,7 @@ metrics, troubleshoot complex performance problems, and make data interoperable
 
 - [**Netdata Agent** or **Agent**](https://github.com/netdata/netdata/blob/master/packaging/installer/README.md): Netdata's distributed monitoring Agent collects thousands of metrics from systems, hardware, and applications with zero configuration. It runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices.
 
-- [**Netdata Cloud** or **Cloud**](https://github.com/netdata/netdata/blob/master/docs/quickstart/infrastructure.md): Netdata Cloud is a web application that gives you real-time visibility for your entire infrastructure. With Netdata Cloud, you can view key metrics, insightful charts, and active alerts from all your nodes in a single web interface.
```

Comment:
@netdata/product and @ktsaou please let me know what to do with https://github.com/netdata/netdata/blob/master/docs/quickstart/infrastructure.md, currently it is out of Learn, it is the only file we didn't have somewhere to put it but we didn't explicitly say to delete it.

Reply:
I think we should have it and on this section
![image](https://github.com/netdata/netdata/assets/82235632/457c18ea-a618-4308-a194-fff2344b57e7)